Sources/NIO/SocketChannel.swift

```diff
@@ -12,7 +12,7 @@
 //
 //===----------------------------------------------------------------------===//
 
-private extension ByteBuffer {
+extension ByteBuffer {
     mutating func withMutableWritePointer(body: (UnsafeMutableRawBufferPointer) throws -> IOResult<Int>) rethrows -> IOResult<Int> {
```

Comment:
Add `private` here please, unless you are actually planning to make this more public. If you are, please add it to `BaseStreamSocketChannel.swift`.

Reply:
@Lukasa we're using this from two places now (`BaseStreamSocketChannel.swift` and `SocketChannel.swift`)